okteto.yml

```diff
@@ -0,0 +1,14 @@
+build:
+  hello-world:
+    image: okteto.dev/node-hello-world:1.0.0
+    context: .
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    image: okteto.dev/node-hello-world:1.0.0
```

Comment:
@gabriel-okteto could you remove the `image` field here? It's not needed. If it's not specified, it uses the same image than the k8s deployment

Reply:
?  we went back and forth on this I think, I am removing it again